src/modules/booking/components/booking-time-select-navigation/booking-time-select-navigation.tsx

```diff
@@ -0,0 +1,54 @@
+import { useMemo, type FC } from 'react';
+import { useSwiper } from 'swiper/react';
+// components
+import { Typography } from '@/modules/core/components/typogrpahy';
+import { Button } from '@/modules/core/components/button';
+// utils
+import { generateDates } from '@/modules/core/utils/date.utils';
+// type
+import type { BookingTimeSelectNavigationProps } from './booking-time-select.interface';
+// style
+import styles from './booking-time-select-navigation.module.scss';
+import { format } from 'date-fns';
+
+const selectDate = (
```

Comment:
Remove this variable and just past it inside the component itself. There is nothing here that I see that needs to be moved outside the component.